scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -25,7 +25,7 @@
             {% else %}
               <td>n.a.</td>
             {% endif %}
-            <td>{{ clinsig.revstat or "n.a." }}</td>
+            <td><div class="text-break">{{ clinsig.revstat or "n.a." }}</div></td>
```

Comment:
This can sometimes be a rather long string nowadays, and no spaces after commas..